src/devices/mercator.ts

```diff
@@ -213,6 +214,43 @@ const definitions: Definition[] = [
             device.save();
         },
     },
+    {
+        fingerprint: [{modelID: 'TS0601', manufacturerName: '_TZE200_agumlajc'}],
+        model: 'SSWMPIR-ZB',
+        vendor: 'Mercator Ikuü',
+        description: 'Combination Sensor',
+        fromZigbee: [fz.occupancy, fz.temperature, fz.humidity, fz.illuminance, fz.on_off, tuya.fz.datapoints],
+        toZigbee: [tuya.tz.datapoints],
+        exposes: [
+            e.occupancy(), e.temperature(), e.humidity(), e.illuminance(), e.light(), 
+            e.enum('state', ea.STATE_SET, ['ON', 'OFF', 'AUTO']).withDescription('Light Mode')
+        ],
+        configure: async (device, coordinatorEndpoint, logger) => {
+            const endpoint = device.getEndpoint(1);
+            await reporting.bind(endpoint, coordinatorEndpoint, ['genBasic', 'genOta', 'genTime', 'genGroups', 'genScenes']);
+            await reporting.bind(endpoint, coordinatorEndpoint, ['genIdentify', 'manuSpecificTuya']);
```

Comment:
Are these binds really needed? Can you check after removing them and re-pairing the device, the device still works?